src/App.vue

```diff
@@ -92,7 +92,7 @@
       </div>
     </transition>
 
-    <SbcHeader />
+    <SbcHeader class="d-flex" />
```

Comment:
To fix header height (same as I did in Filings UI).

Reply:
NB: I also fixed this in sbc-common-components but it's not published yet.

Ref: https://github.com/bcgov/sbc-common-components/pull/303